ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15-periodics.yaml

```diff
@@ -9174,17 +9174,6 @@ periodics:
     job-release: "4.15"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-aws-ipi-f28-longduration-hive-remaining
-  reporter_config:
-    slack:
-      channel: '#forum-prow-hive-qe-ci'
```

Comment:
Those removal are unexpected.

Reply:
It's not from me, I ran `make update`, these contents are removed.

Reply:
Get it fixed manually.